src/utils/filterUniqueKeys.ts

```diff
@@ -1,3 +1,4 @@
+// @debt remove, it seems it is unused
```

Comment:
can we remove it now or should we do it later on?

Reply:
For next PR, didn't check if/where it is used, so I didn't want to risk it.